opengever/maintenance/scripts/update_inbox_group.py

```diff
@@ -0,0 +1,99 @@
+# A script to migrate role assignments of tasks and related documents and
```

Comment:
this looks like a module level docstring thus it should be enclosed by `"""` not by single line comments.